circle.yml

```diff
@@ -14,6 +14,11 @@ dependencies:
     # Set python version
     - pyenv local 3.4.3
 
+    # Install NLTK
+    - pip3 install -U nltk
+    # Install NLTK data
+    - python3 -m nltk.downloader punkt maxent_treebank_pos_tagger averaged_perceptron_tagger
+
```

Comment:
We don't need this commit anymore, nltk data will be loaded from cache_directories :)